proxies/ssh/ssh_recorder.go

```diff
@@ -68,7 +72,7 @@ func (r *SSHRecorder) NewSession(c *proxies.ProxyConn) *SSHRecorderSession {
 // Connect records the connect operation for the underline ssh connection.
 func (rs *SSHRecorderSession) Connect() {
 	rs.r.Channel.Send(
-		message.EventCategoryType(
+		event.Apply(
```

Comment:
New generates an event object directly with the options, why apply does not. It expects to be supplied the Event to be used and applies the option. Useful for cases where we already have a function returning an Event but want to add more details to without manually calling each function individually.